client/core/notification.go

```diff
@@ -100,20 +100,20 @@ type Notification interface {
 // FeePaymentNote is a notification regarding registration fee payment.
 type FeePaymentNote struct {
 	db.Notification
-	ConfirmationsRequired uint32 `json:"confirmationsrequired,omitempty"`
-	Confirmations         uint32 `json:"confirmations,omitempty"`
+	Confirmations *uint32 `json:"confirmations,omitempty"`
+	Dex           string  `json:"dex,omitempty"`
 }
 
-func newFeePaymentNote(subject, details string, severity db.Severity) *FeePaymentNote {
+func newFeePaymentNote(subject, details string, severity db.Severity, dexAddr string) *FeePaymentNote {
 	return &FeePaymentNote{
-		Notification: db.NewNotification("fee payment", subject, details, severity),
+		Notification: db.NewNotification("feepayment", subject, details, severity),
+		Dex:          addrHost(dexAddr),
```

Comment:
@chappjc thanks for the heads up. I updated the code to use the dex host instead. The `feepayment` is also sending the host by making sure to transform `dexAddr` using the `addrHost` helper.